configs/config.dell_optiplex_9010_sff_uefi

```diff
@@ -0,0 +1,47 @@
+CONFIG_USE_OPTION_TABLE=y
+CONFIG_USE_BLOBS=y
+CONFIG_VENDOR_DELL=y
+CONFIG_BOARD_DELL_OPTIPLEX_9010=y
+CONFIG_ONBOARD_VGA_IS_PRIMARY=y
+# CONFIG_DRIVERS_UART_8250IO is not set
+CONFIG_PCIEXP_CLK_PM=y
+CONFIG_HAVE_EM100_SUPPORT=y
+CONFIG_LOCALVERSION="v0.1.0"
+CONFIG_PAYLOAD_EDK2=y
+CONFIG_EDK2_BOOTSPLASH_FILE="3rdparty/dasharo-blobs/dasharo/evaluation_logo.bmp"
+CONFIG_EDK2_BOOT_MENU_KEY=0x0011
+CONFIG_EDK2_GOP_DRIVER=y
+CONFIG_EDK2_REPOSITORY="https://github.com/Dasharo/edk2.git"
+CONFIG_EDK2_TAG_OR_REV="origin/rebased"
+CONFIG_EDK2_SETUP_MENU_KEY=0x000C
+# CONFIG_EDK2_CBMEM_LOGGING=y
+CONFIG_EDK2_FOLLOW_BGRT_SPEC=y
+CONFIG_EDK2_SD_MMC_TIMEOUT=1000
+CONFIG_TPM_MEASURED_BOOT=y
+CONFIG_VENDOR_DELL=y
+CONFIG_BOOTSPLASH_FILE="dasharo-black.bmp"
+CONFIG_BOOTSPLASH_IMAGE=y
```

Comment:
Remove it, UEFI Payload will display the logo.

Reply:
Fixed in [this commit](https://github.com/Dasharo/coreboot/pull/541/commits/a4143cefd7ede989e03dd623ba3e1e8c288f2775)